src/components/TimerBox.js

```diff
@@ -62,6 +63,11 @@ class TimerBox extends Component {
         }, 1000);
     }
 
+    playAlarm() {
+        const alarm = new Audio('http://www.orangefreesounds.com/wp-content/uploads/2017/09/Clock-ringing.mp3');
```

Comment:
will there be any error handling here? What if this file does not exist? Also, there is no mention of long this would play

Reply:
you should initialize alarm inside app.js and pass it as prop to TimerBox.js.